code/modules/mob/living/simple_animal/hostile/vatbeast.dm

```diff
@@ -0,0 +1,107 @@
+///Vatbeasts are creatures from vatgrowing and are literaly a beast in a vat, yup. They are designed to be a powerful mount roughly equal to a gorilla in power.
+/mob/living/simple_animal/hostile/vatbeast
+	name = "vatbeast"
+	desc = "A strange molluscoidal creature carrying a busted growing vat.\nYou wonder if this burden is a voluntary undertaking in order to achieve comfort and protection, or simply because the creature is fused to its metal shell?"
+	icon = 'icons/mob/vatgrowing.dmi'
+	icon_state = "vat_beast"
+	icon_dead = "vat_beast_dead"
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	mob_size = MOB_SIZE_LARGE
+	gender = NEUTER
+	environment_smash = ENVIRONMENT_SMASH_STRUCTURES
+	speak_emote = list("roars")
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	var/obj/effect/proc_holder/tentacle_slap/tentacle_slap
```

Comment:
Add new variables at the bottom of the object definition so we can spot them more easily.
Qdel and null it on `Destroy()`

Reply:
ok